beebop/app.py

```diff
@@ -337,18 +337,30 @@ def get_results(result_type) -> json:
                                          storage_location)
 
 
-def get_clusters_internal(p_hash: str, storage_location: str) -> json:
+def get_clusters_internal(p_hash: str, storage_location) -> dict:
```

Comment:
Pulled this out as a separate method to be used from the existing clusters json endpoint and the project endpoint